larq/quantizers_test.py

```diff
@@ -202,3 +202,38 @@ def test_magnitude_aware_sign():
     np.testing.assert_allclose(
         grad.numpy(), np.where(abs(a) < 1, np.ones(a.shape) * scale_vector, 0)
     )
+
+
+@pytest.mark.parametrize(
+    "fn", [lq.quantizers._dorefa_k_bit_quantizer, lq.quantizers.DorefaQuantizer(2)]
+)
+def test_dorefa_quantize(fn):
+    x = tf.keras.backend.placeholder(ndim=2)
+    f = tf.keras.backend.function([x], [fn(x)])
+    real_values = np.random.uniform(-2, 2, (2, 5))
+    result = f([real_values])[0]
+    k_bit = 2
+    n = 2 ** k_bit - 1
+    assert not np.any(result > 1)
+    assert not np.any(result < -1)
```

Comment:
Shouldn't this be:
```suggestion
    assert not np.any(result < 0)
```

Reply:
No, actually this quantizer only outputs values in the [0,1] interval. It sounds a bit odd but works... s

Reply:
> No, actually this quantizer only outputs values in the [0,1] interval.

Yes, but the above test only checks that there are no values smaller than -1 instead of it should assert that we don't have any values smaller 0. Or am I missing something?

Reply:
No, I just was confused about the colors I guess. You're right of course.